gulpfile.js/config.json

```diff
@@ -25,12 +25,7 @@
       "entries": {
         "app": ["./app.js"]
       },
-      "extensions": ["js", "json"],
-      "babel": {
-        "presets": ["es2015", "stage-1"],
```

Comment:
Jesteś pewien, że nie potrzebujemy tego? Dla wstecznego supportu JS?

Reply:
Nie usunąłem ale przesunąłem to do pliku .babelrc, więc to dalej mamy bo ten support jest wymagany.
Ekipa z babel wywaliła też "es2015" i dała "env" z paczką: babel-preset-env. Więcej info "https://babeljs.io/env/"